api/nakadi-event-bus-api.yaml

```diff
@@ -1647,6 +1695,18 @@ definitions:
         type: string
         description: |
           Offset of the event being pointed to.
+      shift:
```

Comment:
@rcillo Yeah, I think it's better to have that only in a separate endpoint. So that users can do [ cursors + shift = shifted-cursors ]
That will also allow not to include `shift` into original cursor definition.
That will allow not to expose that shift stuff to all usual users.